src/cmdstan/arguments/arg_log_prob_unconstrained_params.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef CMDSTAN_ARGUMENTS_ARG_LOG_PROB_UNCONSTRAINED_PARAMS_HPP
+#define CMDSTAN_ARGUMENTS_ARG_LOG_PROB_UNCONSTRAINED_PARAMS_HPP
+
+#include <cmdstan/arguments/singleton_argument.hpp>
+
+namespace cmdstan {
+
+class arg_log_prob_unconstrained_params : public string_argument {
+ public:
+  arg_log_prob_unconstrained_params() : string_argument() {
+    _name = "unconstrained_params";
+    _description = "Input file of parameter values on unconstrained scale";
```

Comment:
please include a short format description (like "JSON" or "R dump") in `_description`